functions/system.bash

```diff
@@ -471,6 +471,26 @@ memory_split() {
   else
     if echo "gpu_mem=16" >> /boot/config.txt; then echo "OK"; else echo "FAILED"; return 1; fi
   fi
+
+  sed -i '/^dtoverlay=vc4-fkms-v3d/d' /boot/config.txt
+}
+
+## disable or enable framebuffer to provide the maximum amount of memory for Linux operations.
+##
+##    use_framebuffer()
+##
+use_framebuffer() {
+  if ! is_pi; then return 0; fi
+  # TODO
+  # menu option
+  sed -i '^[[:space:]]*max_framebuffers' /boot/config.txt
```

Comment:
What is this sed statement doing? all it looks like is it is finding the line, not performing any operations on it right? Don't you want to delete this line instead so that you don't end up with a whole bunch of duplicated on and off switches of this option?

Reply:
Need this line to delete any occurences. The next lines then re-add it.
well spotted, fixed